docs/data_engineers/accessing_data/elt_pipeline/overview.md

```diff
@@ -9,60 +9,14 @@ supported by [Airbyte](https://airbyte.com/). With the [ELT](https://en.wikipedi
 format, data analysts and engineers can now confidently source KYVE data without worrying about its validity or
 reliability.
 
-The sections that follow were created to help you set up a local environment for testing KYVE's Data Pipeline.
+The sections that follow were created to help you understand how to setup the KYVE source connector of Airbyte in order to retrieve data. There are two options that you can test the ELT connector of KYVE.
 
-## System Requirements
+## Airbyte Local Deployment
 
-To experiment with the ELT pipeline via [Airbyte](https://airbyte.com/) let's first ensure your computer is configured correctly.
+For users that wants to experiment locally with the KYVE data it is recommended the Local Deployment approach of Airbyte. This option requires some configuration steps in order to run locally the Airbyte app. Detailed instructions are presented in [Airbyte Local Deployment](airbyte_local_deployment) section.
 
-### Windows
+## Airbyte Cloud
 
-For Windows users, the Windows Subsystem for Linux 2 (WSL2) must be installed. You must be running Windows 10 version
-2004 or higher (build 19041 or higher), or Windows 11 to use the commands in this tutorial.
+You can connect to the [Airbyte Cloud](https://airbyte.com/airbyte-cloud) platform and try the [KYVE Source Connector](https://cloud.airbyte.com/workspaces/2a2bfca3-c289-48af-a11d-1a9ad69ea187/source/new-source/60a1efcc-c31c-4c63-b508-5b48b6a9f4a6).
```

Comment:
Is this workspace publicly available? The link doesn't work for me.

Reply:
You were right this link is not working for every user, so we will not use a direct link.